src/inert.js

```diff
@@ -89,9 +88,9 @@ class InertRoot {
     }
     this._rootElement = null;
 
-    for (let inertNode of this._managedNodes) {
+    this._managedNodes.forEach(function(inertNode) {
       this._unmanageNode(inertNode.node);
-    }
+    }.bind(this));
```

Comment:
yeah i was thinking of using arrow functions but avoided it because I'd like as much of this to avoid transpilation as possible. I'll definitely see about adding the 2nd argument for `this`, that seems like a nice improvement.

Reply:
+1 for `thisArg`